src/amber/router/pipe/cors.cr

```diff
@@ -5,12 +5,12 @@ module Amber
       property allow_origin, allow_headers, allow_methods, allow_credentials,
         max_age
 
-      def initialize
-        @allow_origin = "*"
-        @allow_headers = "accept, content-type"
-        @allow_methods = "GET, HEAD, POST, DELETE, OPTIONS, PUT, PATCH"
-        @allow_credentials = false
-        @max_age = 0
+      def initialize(
+        @allow_origin = "*",
+        @allow_headers = "accept, content-type",
+        @allow_methods = "GET, HEAD, POST, DELETE, OPTIONS, PUT, PATCH",
```

Comment:
Can you please make the following change

```
ALLOW_METHODS = %w(GET HEAD POST DELETE OPTIONS PUT PATCH)
ALLOW_HEADERS = %w(accept content-type)

@allow_headers = ALLOW_HEADERS,
@allow_methods = ALLOW_METHODS
```

This change should address the following issue https://github.com/amberframework/amber/issues/540